app/controllers/admin/administrators/administrators_controller.rb

```diff
@@ -49,6 +49,9 @@ def create
           AdminProfile.create!(user: user)
         end
         session.delete(:administrator_user)
+
+        set_success_message(heading: "User added", content: "They have been sent an email to sign in")
```

Comment:
Is this true? Or make belief?

Reply:
Have you double checked the message with Shani?

Reply:
This is Shani's message. Alfred is implementing this. Once we know when we will be sending emails to lead providers I will get some updated copy for that one